appveyor.yml

```diff
@@ -7,20 +7,30 @@ environment:
     PYTHONUNBUFFERED: 1
 
   matrix:
+    - PYTHON: "C:\\Miniconda36-x64"
+      CONDA_PY: "27"
+      ARCH: "64"
     - PYTHON: "C:\\Miniconda36-x64"
       CONDA_PY: "35"
       ARCH: "64"
+    - PYTHON: "C:\\Miniconda36-x64"
+      CONDA_PY: "36"
+      ARCH: "64"
 
 install:
   - set PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%
+  - conda config --set always_yes yes --set changeps1 no
   - conda config --add channels omnia
   - conda config --add channels conda-forge
   - conda config --add channels mosdef
   - conda update -yq --all
   - conda install -yq conda-build jinja2
+  - conda create -n test-environment python=%PYTHON_VERSION% --file requirements-dev.txt
+  - activate test-environment
+  - pip install git+https://github.com/mosdef-hub/mbuild.git
+  - pip install -e .
 
 build: false
 
 test_script:
-  - "%CMD_IN_ENV% activate base"
```

Comment:
Any reason why you removed this? I don't recall the specifics but I had to add this at one point to un-break something #195